tests/test_reflect.py

```diff
@@ -38,6 +38,15 @@ def test_hierharchy_arraylist(self):
         self.assertEqual(d["java.lang.Object"], maxLevel)
         self.assertEqual(d["java.util.ArrayList"], 0)
 
+    def test_class(self):
+        lstClz = autoclass("java.util.List")
```

Comment:
that's a nothingbug but in python the standin for "class" is usually "cls" (and sometime "klass" but i'm less fond of that). but here it could even be the more idiomatic "list_class".

Reply:
fair. naming was not very pythonic.